tests/integration_tests/embeddings/test_vertexai.py

```diff
@@ -0,0 +1,29 @@
+"""Test Vertex AI API wrapper.
+In order to run this test, you need to install VertexAI SDK (that is is the private
+preview)  and be whitelisted to list the models themselves:
+export SDK=google_cloud_aiplatform-1.25.dev20230413+language.models-py2.py3-none-any.whl
```

Comment:
Please update this text. The Vertex LLM SDK is now public.

Reply:
done